src/core/Table/TableRowMemoized.tsx

```diff
@@ -48,12 +54,26 @@ const TableRow = <T extends Record<string, unknown>>(props: {
 
   const expandedHeight = React.useRef(0);
 
+  const mergedProps = {
+    ...row.getRowProps({
+      className: cx('iui-row', {
+        'iui-selected': row.isSelected,
+        'iui-row-expanded': row.isExpanded && subComponent,
+        'iui-disabled': isRowDisabled,
+      }),
+    }),
+    ...rowProps?.(row.original),
+  };
+
+  const refs = useMergedRefs(rowRef, mergedProps.ref ?? (() => {}));
```

Comment:
Our hook does not handle undefined ref?

Reply:
It doesn't. I can add to handle that if we will need the same in the future components.

Reply:
hmm.. that empty func looks not nice here.

Reply:
Handling undefined/null.